src/components/Tooltip/index.test.js

```diff
@@ -13,28 +12,4 @@ describe('Tooltip component', () => {
     expect(wrapper).toMatchSnapshot();
   })
 
-  it('sets the right default width', () => {
-    const tooltipMaxWidth = shallow(
-      <Tooltip
-        trigger={<Trigger />}
-        message="tooltip with default max width"
-      />
-    ).find(SELECTOR_TOOLTIP).prop('style').maxWidth;
-
-    expect(tooltipMaxWidth).toBe(DEFAULT_WIDTH);
-  });
-
-  it('overrides default width with `maxWidth` is passed in', () => {
-    const evil = 666;
-    const tooltipMaxWidth = shallow(
-      <Tooltip
-        trigger={<Trigger />}
-        message="tooltip with evil max width"
-        maxWidth={evil}
-      />
-    ).find(SELECTOR_TOOLTIP).prop('style').maxWidth;
-
-    expect(tooltipMaxWidth).toBe(`${evil}px`);
```

Comment:
max width is non critical IMO to the core of the component which is why I've axed this for now (and it's the sort of thing that would be better to visual snapshot than sniff computed widths and stuff)